### PR TITLE
Add tabs to Getting Started and update Android Studio steps

### PR DESCRIPTION
**Pull request by hramos**
## Motivation

There are two ways to get started with React Native: using CRNA, or by building native code. The latter requires a set of dependencies that also need to be installed even by people who use CRNA if they have ejected. 

There needs to be a clear demarcation between the two approaches as well as clear instructions as to which steps are of interest to CRNA using parties or people adding RN to an existing app.

Separating the two approaches entirely by using tabs can achieve this purpose.

## Test Plan

Built and tested the website on Chrome on macOS. Have not yet tested other browsers or OS configurations.

### Quick Start

screencapture-localhost-8079-react-native-docs-getting-started-html-1494974542733

### Building Projects with Native Code

screencapture-localhost-8079-react-native-docs-getting-started-html-1494974580882

## Next Steps

This is a work in progress. Feedback is appreciated. I am not yet happy with the different sets of tabs/buttons, for example.

### CODE REVIEW

**Comment by hramos** [comment]
cc @dikaiosune @ericvicenti

File: docs/GettingStarted.md [hunk 1]
```diff
@@ -7,20 +7,151 @@ permalink: docs/getting-started.html
...
+
+```
+choco install nodejs.install
+```
+
+> You can find additional installation options on [Node's Downloads page](https://nodejs.org/en/download/).
+
+<block class="quickstart linux ios android" />
+
+Follow the [installation instructions for your Linux distribution](https://nodejs.org/en/download/package-manager/) to install Node 4 or newer.
```
Comment (dikaiosune) [inline docs/GettingStarted.md]: I believe CRNA & its deps rely on node 6 or newer

File: docs/GettingStarted.md [hunk 2]
```diff
@@ -35,90 +166,59 @@ Once you've created your project and opened it in the Expo client app, you can p
...
 
-<block class="linux windows ios" />
+<block class="native linux windows ios" />
 
 ## Unsupported
 
-<div>Unfortunately, Apple only lets you develop for iOS on a Mac. If you want to build an iOS app but you don't have a Mac yet, you can try starting with the <a href="" onclick="display('platform', 'android')">Android</a> instructions instead.</div>
+<blockquote><p>A Mac is required to build projects with native code for iOS. You can follow the <a href="docs/getting-started.html" onclick="display('guide', 'quickstart')">Quick Start</a> to learn how to build your app using Create React Native App instead.</p></blockquote>
 
-<center><img src="img/react-native-sorry-not-supported.png" width="150"></img></center>
```
Comment (dikaiosune) [inline docs/GettingStarted.md]: The sad owl was great but I think this is a good call :D

Comment (hramos) [inline docs/GettingStarted.md]: Sad owl is now happy because it learned it can develop iOS apps on Windows.

**Comment by hramos** [comment]
Phew. Spent some more time going through all the steps on Windows and macOS to ensure we are not missing anything on either platform. I've also updated the Quick Start with some next steps similar to the native builds guide. Hopefully it still qualifies as a Quick Start.

# Screenshots

Expand the following to see screenshots:

<details>

## Quickstart
gettingstarted-quickstart

## Native Builds: iOS on macOS
gettingstarted-macos-ios

## Native Builds: iOS on Windows
gettingstarted-windows-ios

## Native Builds: iOS on Linux
gettingstarted-linux-ios

## Native Builds: Android on macOS
gettingstarted-macos-android

## Native Builds: Android on Windows
gettingstarted-windows-android

## Native Builds: Android on Linux
gettingstarted-linux-android

</details>

**Pull request closed without merging**